cli/src/main/scala/com/codacy/analysis/cli/command/CLIApp.scala

```diff
@@ -174,8 +169,8 @@ final case class Analyze(
   @ValueDescription(
     "Reduce issue severity by one level, for non-security issues, for compatibility with GitHub's code scanning. Use in conjunction with `--format sarif`")
   ghCodeScanningCompat: Int @@ Counter = Tag.of(0),
-  @Recurse
-  extras: ExtraOptions)
+  @Hidden // left for backward compatibility
+  analyser: String = "")
```

Comment:
This is to make the tool not fail if you are running `--analyser codacy-plugins`